whois-nrtm4/src/test/java/net/ripe/db/nrtm4/AbstractNrtmIntegrationTest.java

```diff
@@ -287,10 +290,10 @@ protected static List<DeltaFileRecord> getDeltaChanges(final String[] records) t
 
     protected static void assertNrtmFileInfo(final String nrtmInfo, final String type, final int version, final String source) throws JSONException {
         final JSONObject jsonObject = new JSONObject(nrtmInfo);
-        assertThat(jsonObject.getInt("nrtm_version"), is(4));
-        assertThat(jsonObject.getString("type"), is(type));
-        assertThat(jsonObject.getString("source"), is(source));
-        assertThat(jsonObject.getInt("version"), is(version));
+        MatcherAssert.assertThat(jsonObject.getInt("nrtm_version"), is(4));
```

Comment:
why this change? isn't better to load just the static method that we want instead whole MatcherAssert?

Reply:
Agree, I fixed it

Reply:
Not always the case though, sometimes it's clearer to use the class name with the static method